### PR TITLE
Pomodoro timer

### PR DESCRIPTION
**Pull request by atharvabhanage02**
Features included in this PR : 

- User can add Timer to a Task
- Start Timer
- Pause Timer
- Reset Timer
- Dynamic Page Title for Timer Page

Files to look for : 

- Timer : `src/components/Timer`
- TimerPage : `src/Pages/TimerPage`

Here is the Deploy Preview : https://deploy-preview-3--be-focused.netlify.app/

### CODE REVIEW

**Review by abhisheksingh98 — APPROVED** [review]
Looks good to me. You can merge.

**Review by sri-rishi — APPROVED** [review]
code looks good, feel free to merge

File: src/Pages/TimerPage/TimerPage.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { useParams } from "react-router-dom";
+import { Navbar } from "../../components/Navbar/Navbar";
+import { Timer } from "../../components/Timer/Timer";
+import { TimerDetails } from "../../components/TimerDetails/TimerDetails";
+import { useTask } from "../../Context/task-context";
+import "./timerPage.css";
+const TimerPage = () => {
```
Comment (ashishkatiyar2404) [inline src/Pages/TimerPage/TimerPage.jsx:7]: pls give 1 line of space btween importing files and your functions

File: src/Pages/TimerPage/timerPage.css [hunk 2]
```diff
@@ -0,0 +1,12 @@
+.timer-with-details {
+  color: black;
+  background-color: var(--tasks-container-bg);
```
Comment (ashishkatiyar2404) [inline src/Pages/TimerPage/timerPage.css:3]: appreciation : good to see you are consistent with color variables

**Review by ashishkatiyar2404 — APPROVED** [review]
suggested some changes pls look into it.
code is well structured and looks good to merge

**Pull request merged**